docs/faq.md

```diff
@@ -1,17 +1,5 @@
 # FAQ
 
```

Comment:
I think it may be useful to place here a relative link to the new file.

Something like following:

```
## Reusing already running Cluster

See: [reuse_cluster](docs/reuse_cluster.md)
```

Reply:
we can link this also from README.md

Reply:
Added references